easybuild/easyconfigs/i/ispc/ispc-1.12.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'Tarball'
+
+name = 'ispc'
+version = '1.12.0'
+
+homepage = 'http://ispc.github.io/ , https://github.com/ispc/ispc/'
+description = """Intel SPMD Program Compilers; An open-source compiler for high-performance
+ SIMD programming on the CPU. ispc is a compiler for a variant of the C programming language,
+ with extensions for 'single program, multiple data' (SPMD) programming.
+ Under the SPMD model, the programmer writes a program that generally appears
+ to be a regular serial program, though the execution model is actually that
+ a number of program instances execute in parallel on the hardware."""
+
+toolchain = SYSTEM
+
+source_urls = [('http://sourceforge.net/projects/ispcmirror/files/v%(version)s', 'download')]
+sources = ['ispc-v%(version)s-linux.tar.gz']
+checksums = ['79decaef42bc5433ab30f5879eb91af14473a42e8b51ec727a97f61ee918497e']
+
+sanity_check_paths = {
+    'files': ["bin/ispc"],
+    'dirs': []
+}
+
+modextrapaths = {'PATH': ''}
```

Comment:
@furstj This is no longer needed now `ispc` is properly in a `bin/` subdirectory?

Reply:
The modextrapaths is removed in latest commit